release/release_tests.yaml

```diff
@@ -593,7 +593,7 @@
   team: ml
 
   cluster:
-    cluster_env: ../horovod_tests/app_config_master.yaml
+    cluster_env: horovod/app_config_master.yaml
```

Comment:
Ah this is my fault! Forgot to update this app config as I was doing DistributedTrainable deprecation. 

Can you use the app config in ../air_tests/horovod instead of creating a new one?

Reply:
I think it's preferable to keep configs in their respective subfolders. Otherwise it will be confusing - if I change something for the air test, I wouldn't usually test the ml user tests, but they could break from that.
Let me know if you feel strongly about this one, I just want to avoid surprises in the future.

Reply:
Got it yeah, that sounds good. The original intention was to have one canonical app config for all horovod tests, but now that we will be running individual release tests as part of PRs what you said makes sense!